yb-voyager/cmd/exportSchema.go

```diff
@@ -27,6 +29,10 @@ import (
 	"github.com/spf13/cobra"
 )
 
+type MetaInfo struct {
+	SourceDBType, SourceDBName, SourceDBSchema, SourceDBVersion string
+}
```

Comment:
Let's name it as `MigInfo` as short for "Migration Info".
One field per line, please.
Move this struct and associated two functions in a separate `miginfo.go` file.